src/coalesce-vue/src/metadata.ts

```diff
@@ -148,6 +151,7 @@ export interface ModelType extends CustomReferenceTypeBase, ApiRoutedType {
 export interface DataSourceType extends Metadata {
   readonly type: "dataSource";
   readonly isDefault?: true;
+  readonly description?: string;
```

Comment:
Unsure about this change - added to fix a test failure.

Reply:
Which test is failing?

Reply:
`TargetClassesFullGenerationTest.VueOutputCompiles`
Specifically the protected method to check TS compilation `TargetClassesFullGenerationTest.AssertVueSuiteTypescriptOutputCompiles` with the message:
:../IntelliTect.Coalesce.CodeGeneration.Tests/out/.NETCoreApp,Version=v8.0/VueSuite/src/metadata.g.ts(330,7): error TS2353: Object literal may only specify known properties, and 'description' does not exist in type 'DataSourceType'.

Reply:
Ah - this is from the empty strings, I think. So with the change from the comment below this code should be removed. Didn't totally understand how these tests worked, but that's a clearer picture now. 
https://github.com/IntelliTect/Coalesce/pull/419#discussion_r1714034389

Reply:
Oh, yeah, that's fine.

Reply:
You can keep it - someone could still put a description attribute on a data source and we don't want that to have type errors (even if we don't currently do anything with hypothetical descriptions on datasources).